Essentials/src/com/earth2me/essentials/Teleport.java

```diff
@@ -247,6 +249,13 @@ protected void respawnNow(IUser teleportee, TeleportCause cause) throws Exceptio
     //The warp function is a wrapper used to teleportPlayer a player to a /warp
     @Override
     public void warp(IUser teleportee, String warp, Trade chargeFor, TeleportCause cause) throws Exception {
+        UserWarpEvent event = new UserWarpEvent(teleportee, warp, chargeFor);
+        Bukkit.getServer().getPluginManager().callEvent(event);
+
+        if(event.isCancelled()){
```

Comment:
This should still be spaced out: `if (event.isCancelled()) {`

Reply:
Ok I will do it tonight